tutorial/urls.py

```diff
@@ -1,12 +1,21 @@
-from snippets import views
-from django.conf.urls import patterns, url, include
-from rest_framework.routers import DefaultRouter
+"""tutorial URL Configuration
```

Comment:
It looks like the change on this file stops the application from running "out of the box".